newIDE/app/src/ObjectsRendering/ObjectsRenderingService.js

```diff
@@ -74,9 +75,21 @@ const ObjectsRenderingService = {
         associatedObjectConfiguration,
         pixiContainer,
         PixiResourcesLoader,
+        // This parameter is only used by RenderedTileMapInstance.
```

Comment:
Can you comment why it's used for?

Reply:
I've checked and the tile map extension doesn't actually need it.
- https://github.com/4ian/GDevelop/pull/4260